lib/stat_tracker.rb

```diff
@@ -18,6 +18,17 @@ def self.from_csv(locations)
     StatTracker.new(games, teams, game_teams)
   end
 
+  def find_team_name_by_id(team_id)
+    Team.find_team_name_by_id(team_id)
+  end
+
+  def highest_scoring_home_team 
+    find_team_name_by_id(GameTeam.avg_scores_per_team_home.max_by {|team_id, goals| goals}.first)
```

Comment:
We may want to refactor these down the road, but nice job getting these figured out! Definitely tough methods.